File: java-api-core/src/com/kinvey/java/store/KinveyLiveServiceCallback.java
package com.kinvey.java.store;

/**
 * Created by yuliya on 2/20/17.
 */

interface KinveyLiveServiceCallback<T> {

    void onNext(T next);

    void onError(Exception e);

    void onStatus(KinveyLiveServiceStatus status);

}

File: java-api-core/src/com/kinvey/java/store/KinveyLiveServiceCallback.kt
package com.kinvey.java.store

/**
 * Created by yuliya on 2/20/17.
 */

interface KinveyLiveServiceCallback<T> {

    fun onNext(next: T)

    fun onError(e: Exception)

    fun onStatus(status: KinveyLiveServiceStatus)

}
